library/psa_crypto.c

```diff
@@ -3964,9 +4062,24 @@ psa_status_t psa_cipher_update( psa_cipher_operation_t *operation,
         goto exit;
     }
 
-    ret = mbedtls_cipher_update( &operation->ctx.cipher, input,
-                                 input_length, output, output_length );
-    status = mbedtls_to_psa_error( ret );
+    if( operation->alg == PSA_ALG_ECB_NO_PADDING )
+    {
+        /* mbedtls_cipher_update has an API inconsistency: it will only
```

Comment:
After moving the bulk of the code to `psa_cipher_update_ecb_internal()`, this comment seems misplaced. Not a blocker.